### ISSUE TITLE
Better solution for <meta> tag in index.js 

### ISSUE THREAD

**Issue by B-Stefan** [duizendnegen/ember-cli-lazy-load#3]
Currently, we insert the a `<meta>` tag in the index.js file to pass the asset-map to the application.  
This is necessary to support the fingerprint addon but if anybody know a better way to pass information from the build step to the application please let me know.

**Comment by MiguelMadero** [comment]
`AssetRev` could potentially also modify the output of config/environment that include the information fo the bundles if the bundle data includes something that looks like a path. Like `assets/bundle.js` instead of just the bundle-name.

**Comment by duizendnegen** [comment]
Yep, that'd be a much better path - especially considering the Ember team themselves are also looking at now embedding config in index.html themselves too.

**Comment by MiguelMadero** [comment]
> embedding config in index.html themselves too.

That works already if you set `storeConfigInMeta: true`  on your cli-build
